cluster/gce/config-default.sh

```diff
@@ -175,7 +175,7 @@ if [[ ${KUBE_ENABLE_INSECURE_REGISTRY:-false} == "true" ]]; then
 fi
 
 # Optional: customize runtime config
-RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-}"
+RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-admissionregistration.k8s.io/v1alpha1}"
```

Comment:
This doesn't seem to be useful, e2e test says
```
I1018 22:29:18.099] [It] Should be able to deny pod creation
I1018 22:29:18.099]   test/e2e/apimachinery/webhook.go:56
I1018 22:29:18.100] Oct 18 22:28:53.647: INFO: dynamic configuration of initializers requires the alpha admissionregistration.k8s.io group to be enabled
```

Reply:
change it to `RUNTIME_CONFIG=admissionregistration.k8s.io/v1alpha1`, or else it will still read in `KUBE_RUNTIME_CONFIG`